sycl/source/detail/config.hpp

```diff
@@ -120,17 +122,20 @@ template <> class SYCLConfig<SYCL_BE> {
     const char *ValStr = BaseT::getRawValue();
     const std::array<std::pair<std::string, backend>, 3> SyclBeMap = {
         {{"PI_OPENCL", backend::opencl},
-         {"PI_LEVEL0", backend::level_zero},
+         {"PI_LEVEL_ZERO", backend::level_zero},
          {"PI_CUDA", backend::cuda}}};
     if (ValStr) {
+      if (strcmp(ValStr, "PI_LEVEL0") == 0) {
```

Comment:
Why did you want to support "PI_LEVEL0" name? Is this to accommodate current users/tests? Please add it to the array  at line 123.

Reply:
yes, for backwards compatibility.  done.